packages/namespaces/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "interop-namespaces",
+  "version": "0.1.0",
+  "description": "> TODO: description",
+  "author": "Ángel Araya <angel.araya@janeirodigital.com>",
```

Comment:
You might want to add yourself here @elf-pavlik

Reply:
We can discuss that tomorrow. We may need to remove author and use contributors since only the later allows an array.